src/js/chat.js

```diff
@@ -252,8 +248,8 @@ function postMessage(raw_msg, isHistory ) {
         return;
```

Comment:
It's not exactly at this point, but could you add a space before the parameters of postMessage so that line 245 reads `function postMessage( raw_msg, isHistory ) {`?